metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -137,6 +143,9 @@ def __init__(self, config: VerticaConfig, ctx: PipelineContext):
         self.projection_lineage_map: Optional[
             Dict[str, List[Tuple[str, str, str]]]
         ] = None
+        self.tables: DefaultDict[str, List[str]] = DefaultDict(list)
```

Comment:
As we discussed on our calls, this logic should live in `vertica-sqlalchemy-dialect`.

Additionally, you've changed the contract for inspector.get_table_properties and similar methods by removing the table name param.

I cannot accept this PR until these issues are resolved.